ni_measurementlink_service/measurement/info.py

```diff
@@ -67,7 +67,7 @@ class TypeSpecialization(enum.Enum):
     """Enum that represents the type specializations for measurement parameters."""
 
     NoType = ""
-    Pin = "pin"
+    IOResource = "ioresource"
```

Comment:
Removing `Pin` breaks source compatibility

Reply:
The TypeSpecialization does appear as a public class, although I tried to search many places for any usage and couldn't find any (besides the internal usages here).

Assuming that it's still important to avoid, and assuming that we still want to end up with "ioresource" on the wire (your other comment), would the following be valid (I'm not familiar with this structure of a class that appears to be an enum that has string values)? Specifically that there are two values that are the same.

```python
class TypeSpecialization(enum.Enum):
    """Enum that represents the type specializations for measurement parameters."""

    NoType = ""
    Pin = "ioresource"
    Path = "path"
    Enum = "enum"
    IOResource = "ioresource"
```

Reply:
I've put Pin back